my_solutions.rb

```diff
@@ -0,0 +1,37 @@
+
+## Janice Lichtman: Solutions to Whiteboard Practice #2
+
+def mirror(list)
+  result = []
+  (0...list.length).each do |n|
+    result << list[n]
+  end
+
+  (0...list.length).each do |n|
+    result << list[list.length - 1 - n]
+  end
+  return result
+end
```

Comment:
Looks good. If you had used indexing instead of the shovel though, you could have done this with one loop.

Still, looks good!